Sources/TuistSupport/Constants.swift

```diff
@@ -9,7 +9,7 @@ public enum Constants {
     public static let githubSlug = "tuist/tuist"
     public static let communityURL = "https://community.tuist.io"
     public static let backboneURL = URL(string: "https://backbone.tuist.io")!
-    public static let version = "2.4.0"
```

Comment:
Looks like we didn't update this in the branch during `2.5.0` release 🤔

Reply:
oof, looks like it 😞 fortunately, it should not be a big deal because the binary should be created by the Github action where this string is updated automatically. However, since the github action cannot push, we then need to update some stuff manually (including this)